wcivf/apps/elections/templates/elections/includes/_cancelled_election.html

```diff
@@ -47,7 +47,7 @@ <h4>{% trans "Cancelled and Rescheduled Election" %}</h4>
 {% else %}
     {% if object.metadata.cancelled_election %}
         <h4>{{ object.metadata.cancelled_election.title }}</h4>
-        <p>{{ object.metadata.cancelled_election.detail }}</p>
+        {{ object.metadata.cancelled_election.detail | linebreaks }}
     {% else %}
         <h4>{% trans "Cancelled Election" %}</h4>
         <p>{% trans "This election was cancelled." %}</p>
```

Comment:
There is still this fallback case here where we're saying "cancelled".
This isn't a case I'd expect to hit often as it only happens if we didn't have a cancelled reason or metadata object, but I guess we can end up here and if we do, we don't know if this election is postponed or cancelled.

Similar to WDIV, the best thing I can think of to say here would be
"This election will not go ahead as planned"
but if you've got any other ideas for generic words, I'm open to ideas on this.

Reply:
>  "This election will not go ahead as planned"

This is a good suggestion, but I'll let @pmk01 decide on what's best. I don't think saying "cancelled" is _so_ bad that we must avoid it at all costs!

Reply:
I am happy with either text! For context the case I am thinking about is this one: https://www.dailyrecord.co.uk/in-your-area/renfrewshire/shambles-key-election-cancelled-under-25547891

Reply:
OK. Lets leave this one as it stands for now